src/components/cap-reporter.js

```diff
@@ -119,7 +119,11 @@ export default class CapReporter extends LitElement {
 						<ul class="reporter__volumeList">
 							<p class="reporter__volumeTitle">Volume number:</p>
 							${this.volumesData
-								.sort((a, b) => a.volume_number - b.volume_number)
+								.sort(
```

Comment:
Wouldn't this allow `33 Suppl. 34` to come before `33`, if it happened to be that way in the input?

Reply:
Yes, that would allow `33 Suppl.` to come before `33` if it happened to be that way in the input. (`34` is it's own beast).

But, we don't have very many of these, and all the ones I checked looked good, so I didn't think it was worth adding a secondary sort.

If you disagree, I'm happy to add one.

Reply:
No, seems fine, go for it.